medusa/search/backlog.py

```diff
@@ -116,13 +104,13 @@ def search_backlog(self, which_shows=None):
             segments = self._get_segments(cur_show, from_date)
 
             for season, segment in iteritems(segments):
-                self.currentSearchInfo = {'title': cur_show.name + " Season " + str(season)}
+                self.currentSearchInfo = {'title': cur_show.name + ' Season ' + str(season)}
 
                 backlog_queue_item = BacklogQueueItem(cur_show, segment)
                 app.search_queue_scheduler.action.add_item(backlog_queue_item)  # @UndefinedVariable
 
             if not segments:
-                logger.log(u"Nothing needs to be downloaded for %s, skipping" % cur_show.name, logger.DEBUG)
+                log.debug(u'Nothing needs to be downloaded for {0}, skipping', cur_show.name)
```

Comment:
@fernandog I dont think adding show is necessary.  If you would like the quotes around it I'm fine with that

`Nothing needs to be downloaded for "Game of Thrones"`